platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -59,15 +63,15 @@ public ExportCommand(ConfigurationMigrationService configurationMigrationService
         this(configurationMigrationService, security, pathBuilder.build());
     }
 
-    ExportCommand(ConfigurationMigrationService configurationMigrationService,
-            Security security, Path path) {
+    ExportCommand(ConfigurationMigrationService configurationMigrationService, Security security,
```

Comment:
I think we're in agreement here; I plan on getting rid of the constructor args in the main class, but is it also confusing in the test class?